client/src/components/viewTicket/index.js

```diff
@@ -137,13 +137,13 @@ export default class ViewTicket extends React.Component {
         )
     }
 
-    renderField = (field) => {
-        return this.renderTicketInfo(field, this.state.ticket.details[field])
+    renderField = (field,index) => {
+        return this.renderTicketInfo(field, this.state.ticket.details[field], index)
     }
 
-    renderTicketInfo = (name, value) => {
+    renderTicketInfo = (name, value, index) => {
         return (
-            <tr>
+            <tr key={index || Math.random()}>
```

Comment:
oh it's not the updates renderer it's the ticket fields one, so `name` actually represents the field name, not a user name (poor naming on my behalf). I'll rename the variable and change accordingly when I merge if that's ok.